modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLSubscriptionSender.java

```diff
@@ -52,13 +55,8 @@ protected Boolean hasSubscribePermission(
 		return super.hasSubscribePermission(permissionChecker, subscription);
 	}
 
-	private static volatile ModelResourcePermission<DLFolder>
-		_dlFolderModelResourcePermission =
-			ServiceProxyFactory.newServiceTrackedInstance(
-				ModelResourcePermission.class, DLSubscriptionSender.class,
-				"_dlFolderModelResourcePermission",
-				"(model.class.name=" + DLFolder.class.getName() + ")", true);
-
-	private long _targetFolderId;
+	private final ModelResourcePermission<DLFolder>
```

Comment:
@W2jx Could you test to serialize and then deserialize an instance of this class? I wonder if this instance field can be serialized, since `ModelResourcePermission` is not a `Serializable`.